main/main.go

```diff
@@ -162,11 +165,27 @@ func reportExecutionStatus() {
 
 func install() {
 	operation := "install"
+	if runtime.GOOS == "linux" {
+		path, _ := os.Getwd()
+		_, err := exec.Command("sudo", filepath.Join(path, "scripts/service_install.sh")).Output()
+		if err != nil {
+			errorMessage := fmt.Sprintf("Error installing service: %+v", err)
+			errorLogger.Println(errorMessage)
+		}
+	}
 	testCommand(operation)
 }
 
 func enable() {
 	operation := "enable"
+	if runtime.GOOS == "linux" {
+		path, _ := os.Getwd()
+		_, err1 := exec.Command("sudo", filepath.Join(path, "scripts/service_enable.sh")).Output()
```

Comment:
Just noticed you're not capturing the output of these commands, might be worthwhile to capture it and log it just to keep track of whatever goes on in the script.

Reply:
None of those commands produce any output so not useful to capture empty log.

Reply:
I'm sure they have some stdout right? And I'm not asking you to create a new log, just add it in the same logger. Even if its empty, it wont do any harm

Reply:
added